packages/aragon-wrapper/src/rpc/handlers/update-forwarded-action.js

```diff
@@ -0,0 +1,15 @@
+export default function (request, proxy, wrapper) {
+  const [
+    actionId,
+    evmScript,
+    state
+  ] = request.params
+
+  wrapper.setForwardedAction(
+    proxy.address,
+    actionId,
+    evmScript,
+    state
+  )
+  return Promise.resolve()
+}
```

Comment:
Can we combine this with `get-forwarded-actions.js` into a `forward-actions.js` with multiple exports (e.g. similar to `external.js`)?